src/program.js

```diff
@@ -251,6 +251,12 @@ Example: $0 --help run.
         demand: false,
         type: 'boolean',
       },
+      'pref': {
+        describe: 'Launch firefox with custom preferences. Lightweight ' +
+                  'alternative to creating custom profile.',
+        demand: false,
+        type: 'object',
```

Comment:
I took a brief look at this alternative `--pref.key.subkey=value` syntax, but I'm unsure about it, in particular if it makes this feature more easily implementable and/or if it makes this command line option more or less readable for the user.

The usage on the command line would make the command to looks like:

```
$ web-ext run --pref.browser.search.region=IT \           
              --pref.general.useragent.locale=it-IT
```

while I was thinking about something like the following:

```
$ web-ext run --pref "browser.search.region=IT" \           
              --pref "general.useragent.locale=it-IT"
```

@kumar303 what do you think about the above two command line syntaxes for this `--pref` option?